src/services/api.ts

```diff
@@ -20,4 +21,12 @@ export const telescoperApi = {
     const response = await this.api.post('/api/users/', { email, username, password })
     return response.data
   },
+  async login(username: string, password: string): Promise<{ token: string }> {
+    const response = await this.api.post('/api/users/signin', { username, password })
```

Comment:
This looks like it should be /api/users/login not /api/users/signin

Reply:
good catch!